README.md

```diff
@@ -14,6 +14,11 @@ ember install ember-cli-blanket
 ember install:addon ember-cli-blanket
 ```
 
+## Requirements
+* If using Mocha, Testem `>= 1.6.0` for which you need ember-cli `> 2.4.3`
```

Comment:
Do you know the history behind this one?  Most of our projects are running `ember-cli@2.3`